api-reference/_hidden/PdfCell/padding/padding.md

```diff
@@ -4,7 +4,212 @@ type: Object
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Generates space around the text of the cell inside the DataGrid. Uses the measure units, which are specified in the constructor for the [jsPDFDocument](/api-reference/50%20Common/Object%20Structures/PdfExportDataGridProps/jsPDFDocument.md '/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument') object.
```

Comment:
Uses the measure units which are specified in the constructor **of** the [jsPDFDocument](/api-reference/50%20Common/Object%20Structures/PdfExportDataGridProps/jsPDFDocument.md '/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument') object.